src/vis/vis.js

```diff
@@ -144,6 +144,10 @@ var VisModel = Backbone.Model.extend({
       vis: this
     });
 
+    this.analysis.findNodeById = function (nodeId) {
```

Comment:
Now that I understand what this is here... And., since this method is currently "public" (for BUILDER and deep-insights.js), do we want to add a very basic test for it?